### PR TITLE
ONL-5436 add tooltip to the CTA dropdown

### PR DESCRIPTION
**Pull request by jaimeperezpedra**
image_2021-01-24_171433

### CODE REVIEW

File: src/components/ec-dropdown/__snapshots__/ec-dropdown.spec.js.snap [hunk 1]
```diff
@@ -3,7 +3,19 @@
 exports[`EcDropdown #slots should forward CTA slot 1`] = `
 <li
   class="ec-dropdown-search__cta-area"
-  data-test="ec-dropdown-search__cta-area"
```
Comment (jaimeperezpedra) [inline src/components/ec-dropdown/__snapshots__/ec-dropdown.spec.js.snap:6]: I'm not sure about this addition on the test that was before, can someone confirm it with me?

File: src/components/ec-dropdown/ec-dropdown.spec.js [hunk 2]
```diff
@@ -151,6 +151,17 @@ describe('EcDropdown', () => {
       });
       expect(wrapper.findByDataTest('ec-dropdown-search__cta-area').element).toMatchSnapshot();
     });
+
+    it('should forward CTA slot and have a tooltip on the CTA', () => {
```
Comment (jmesao) [inline src/components/ec-dropdown/ec-dropdown.spec.js:155]: @jaimeperezpedra Could we say -> should forward CTA slot with tooltip?

File: src/components/ec-dropdown/ec-dropdown.story.js [hunk 3]
```diff
@@ -180,6 +180,7 @@ stories.add('all', () => ({
             label="Single value - with CTA/Loading/Search"
             placeholder="Single value - with CTA/Loading/Search"
             :is-loading="true"
+            tooltip-cta="Radon tooltip cta"
```
Comment (MaruaEl) [inline src/components/ec-dropdown/ec-dropdown.story.js:183]: "Random" 😜

**Review by matallo — APPROVED** [review]
Small comments from other members

File: src/components/ec-dropdown/ec-dropdown.spec.js (hunk 2 above)
Comment (jaimeperezpedra) [reply src/components/ec-dropdown/ec-dropdown.spec.js]: done!

File: src/components/ec-dropdown/ec-dropdown.story.js (hunk 3 above)
Comment (jaimeperezpedra) [reply src/components/ec-dropdown/ec-dropdown.story.js]: done!

**Comment by jaimeperezpedra** [comment]
I change the size of the icon of the currency component after some conversation with Uve, doing it the same size as the label icon on the input field

**Pull request merged**